### PR TITLE
Bug 1218314 - Re-skin and animate UI components in the Firefox Account sign in flow

### PR DESCRIPTION
**Pull request by begeeben**

### CODE REVIEW

File: tv_apps/smart-system/fxa/js/screens/fxam_refresh_auth.js [hunk 1]
```diff
@@ -34,6 +35,12 @@ var FxaModuleRefreshAuth = (function() {
     FxaModuleUI.disableDoneButton();
   }
 
+  function _togglePasswordVisibility() {
+    /*jshint validthis:true*/
+    var passwordFieldType = !!this.fxaShowPwRefresh.checked ? 'text' : 'password';
```
Comment (danhuang1202) [inline tv_apps/smart-system/fxa/js/screens/fxam_refresh_auth.js]: jshint check failed: Line is too long.

File: tv_apps/smart-system/fxa/js/screens/fxam_refresh_auth.js [hunk 2]
```diff
@@ -96,6 +103,23 @@ var FxaModuleRefreshAuth = (function() {
         }
       });
 
+      this.fxaPwInputRefresh.addEventListener('focus', () => {
+        setTimeout(this.fxaPwInputRefresh.select.bind(this.fxaPwInputRefresh));
+      });
+
+      this.fxaShowPwRefresh.addEventListener('keypress', e => {
+        if (e.keyCode && e.keyCode === KeyEvent.DOM_VK_RETURN) {
```
Comment (danhuang1202) [inline tv_apps/smart-system/fxa/js/screens/fxam_refresh_auth.js]: jshint check failed: 'KeyEvent' is not defined

File: tv_apps/browser/style/components.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  display: flex;
+  align-items: center;
+}
+
+input[type="checkbox"].settings-list-checkbox {
+  display: none;
+}
+
+input[type="checkbox"].settings-list-checkbox + label {
+  display: inline-block;
```
Comment (danhuang1202) [inline tv_apps/browser/style/components.css]: the checked icon in checkbox is not at the vertical center in browser setting page. I think may use following css to do theese 
{
display: inline-flex;
align-items: center;
justify-content: center;
}

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=302464b6fcdc33fdbfd514ef2f94bc0aef740bfc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=123a30e7e9e295a8457d4f0c7a7030c8f94ffbe6

**Pull request merged**